FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,6 +5,30 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
+	typealias RemoteFeedLoaderResult = Swift.Result<Data, Error>
```

Comment:
`RemoteFeedLoaderResult` is already inside the scope of `RemoteFeedLoader`, so there's no need to repeat the `RemoteFeedLoader` part in its name. 

Also, it's actually not the expected `Result` from `Remote`, it's a partial step into creating the expected `Swift.Result<[FeedImage], Error>`. A better name would describe this as the resulting `Data`, and we can also use the default `Swift.Error` to make it easier to map from `httpClientResult` (as shown in a later comment):
```suggestion
	typealias DataResult = Swift.Result<Data, Swift.Error>
```